src/keycloak-theme/login/pages/register/register.component.html

```diff
@@ -1,74 +1,183 @@
+        <div [class]="'kcInputWrapperClass' | kcClass">
+          <input
+            type="password"
+            id="password"
+            [class]="'kcInputClass' | kcClass"
+            name="password"
+            autoComplete="new-password"
+          />
         </div>
-    </div>
+      </div>
 
-    <div [class]="('kcFormGroupClass' | kcClass) + ' ' + (kcContext.messagesPerField.printIfExists('password-confirm', ('kcFormGroupErrorClass')) | kcClass)">
-        <div [class]="('kcLabelWrapperClass' | kcClass)">
-            <label for="password-confirm" [class]="('kcLabelClass' | kcClass)">Passwort bestätigen</label>
+      <div
+        [class]="
+          ('kcFormGroupClass' | kcClass) +
+          ' ' +
+          (kcContext.messagesPerField.printIfExists(
+            'password-confirm',
+            'kcFormGroupErrorClass'
+          ) | kcClass)
+        "
+      >
+        <div [class]="'kcLabelWrapperClass' | kcClass">
+          <label for="password-confirm" [class]="'kcLabelClass' | kcClass"
+            >Passwort bestätigen</label
+          >
         </div>
-        <div [class]="('kcInputWrapperClass' | kcClass)">
-            <input type="password" id="password-confirm" [class]="('kcInputClass' | kcClass)" name="password-confirm" />
+        <div [class]="'kcInputWrapperClass' | kcClass">
+          <input
+            type="password"
+            id="password-confirm"
+            [class]="'kcInputClass' | kcClass"
+            name="password-confirm"
+          />
         </div>
-    </div>
+      </div>
     }
 
-    <div [class]="('kcFormGroupClass' | kcClass)">
-        <div id="kc-form-options" [class]="('kcFormOptionsClass' | kcClass)">
-            <div [class]="('kcFormOptionsWrapperClass' | kcClass)">
-                <span><a [href]="kcContext.url.loginUrl">Zurück zum Login</a></span>
-            </div>
-        </div>
-        <div id="kc-form-buttons" [class]="('kcFormGroupClass' | kcClass)">
-            <input type="hidden" id="id-hidden-input" name="credentialId" [value]="kcContext.auth?.selectedCredential" />
-            <input tabIndex="4"
-              [class]="('kcButtonClass' | kcClass) + ' ' + ('kcButtonPrimaryClass' | kcClass) + ' ' + ('kcButtonBlockClass' | kcClass) + ' ' + ('kcButtonLargeClass' | kcClass)"
-              name="login" id="kc-login" type="submit" value="Registrieren" [disabled]="kcContext.isRegisterButtonDisabled" />
+    <div [class]="'kcFormGroupClass' | kcClass">
+      <div id="kc-form-options" [class]="'kcFormOptionsClass' | kcClass">
+        <div [class]="'kcFormOptionsWrapperClass' | kcClass">
+          <span><a [href]="kcContext.url.loginUrl">Zurück zum Login</a></span>
         </div>
+      </div>
+      <div id="kc-form-buttons" [class]="'kcFormGroupClass' | kcClass">
+        <!-- <input
+          type="hidden"
+          id="id-hidden-input"
+          name="credentialId"
+          [value]="kcContext.auth?.selectedCredential"
```

Comment:
Thanks to type check I noticed that here selectedCredential doesn't exist on register.ftl

Reply:
yep i missed it in merge conlict, my bad